packages/react-scripts/scripts/start.js

```diff
@@ -11,6 +11,11 @@
 
 process.env.NODE_ENV = 'development';
 
+// Load environment variables from .env file. Surpress warnings using silent
+// if this file is missing. dotenv will never modify any environment variables
+// that have already been set.
+require('dotenv').config({silent: true});
```

Comment:
> but I can't get it to work (using NODE_PATH)

I thought this would only be used to specify the [`REACT_APP_*` environment variables](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-custom-environment-variables) that we officially support? But there's some other use case for this that I'm missing? Can you clarify what you mean by "using NODE_PATH"?